plugin/db/pg/sync.go

```diff
@@ -493,12 +494,12 @@ func getViews(txn *sql.Tx) ([]*viewSchema, error) {
 		if err := rows.Scan(&view.schemaName, &view.name, &def); err != nil {
 			return nil, err
 		}
-		// Return error on NULL view definition.
+		// Log error on NULL view definition.
 		// https://github.com/bytebase/bytebase/issues/343
-		if !def.Valid {
-			return nil, fmt.Errorf("schema %q view %q has empty definition; please check whether proper privileges have been granted to Bytebase", view.schemaName, view.name)
+		log.Warn(fmt.Sprintf("schema %q view %q has empty definition; please check whether proper privileges have been granted to Bytebase", view.schemaName, view.name))
```

Comment:
If the view had non-empty definition, we should log it?

Reply:
Make some edits.